### PR TITLE
feat: Add searchbar to top of conversation list (WPB-5949)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5949" title="WPB-5949" target="_blank">StoryWPB-5949</a>  Add searchbar to top of conversation list
  </td></table>
  <br />
 

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx [hunk 1]
```diff
@@ -315,27 +322,88 @@ const Conversations: React.FC<ConversationsProps> = ({
...
+              startContent={
+                <SearchIcon
+                  width={14}
+                  height={14}
+                  css={{
+                    top: '50%',
+                    left: 10,
+                    position: 'absolute',
+                    transform: 'translateY(-50%)',
+                  }}
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx]: Please move all of styles to Conversations.styles.ts :)

Comment (thisisamir98) [reply src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx]: done in https://github.com/wireapp/wire-webapp/pull/16676/commits/b7d6961875429465236edae9c665391f98052705

File: src/script/page/LeftSidebar/panels/Conversations/Conversations.styles.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  height: '32px',
+  borderRadius: 8,
+  paddingLeft: 36,
+};
+
+export const searchInputWrapperStyles: CSSObject = {
+  marginBottom: 12,
+  marginTop: 12,
+  paddingLeft: 16,
+  paddingRight: 16,
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/Conversations/Conversations.styles.ts]: You can use for margin-top/bottom - marginBlock: 12
And for padding - paddingInline: 16 ;)

Comment (thisisamir98) [reply src/script/page/LeftSidebar/panels/Conversations/Conversations.styles.ts]: https://github.com/wireapp/wire-webapp/pull/16676/commits/173fe223657c45c91bf66b0c62aad4f72b9144a5

**Review by przemvs — APPROVED** [review]

**Pull request merged**